### PR TITLE
feat(events): instrument Increase/Decrease-Operator-Stake PE-6854

### PR DESCRIPTION
**Pull request by arielmelendez**

### CODE REVIEW

File: src/main.lua [hunk 1]
```diff
@@ -838,31 +838,43 @@ Handlers.add(
...
 			})
+		end, gar.increaseOperatorStake, msg.From, quantity)
+		if not shouldContinue2 then
+			return
 		end
+
+		msg.ioEvent:addField("Sender-New-Balance", balances[msg.From])
+		if gateway ~= nil then
+			msg.ioEvent:addField("GW-New-Operator-Stake", gateway.operatorStake)
+			msg.ioEvent:addField("GW-Previous-Operator-Stake", gateway.operatorStake - quantity)
```
Comment (dtfiedler) [inline src/main.lua:869]: nit: add
```
msg.ioEvent:addField(`Gateway`, msg.From) -- or add `gateway.gatewayAddress` as an attribute on the returned gateway object
```
and remove the `GW` prefex in the provided tags

File: src/main.lua [hunk 2]
```diff
@@ -877,31 +889,54 @@ Handlers.add(
...
+		if not shouldContinue2 then
+			return
 		end
+
+		msg.ioEvent:addField("Sender-New-Balance", balances[msg.From]) -- should be unchanged
+		if gateway ~= nil then
+			local previousStake = gateway.operatorStake + quantity
+			msg.ioEvent:addField("GW-New-Operator-Stake", gateway.operatorStake)
+			msg.ioEvent:addField("GW-Previous-Operator-Stake", previousStake)
+			msg.ioEvent:addField("GW-Vaults-Count", #(gateway.vaults or {}))
```
Comment (dtfiedler) [inline src/main.lua:922]: see same comment as above

File: src/main.lua (hunk 1 above)
Comment (arielmelendez) [reply src/main.lua:869]: From is auto-added to every event. Would it be important to you to see it also added to a `Gateway` field?

File: src/main.lua (hunk 2 above)
Comment (arielmelendez) [reply src/main.lua:922]: `GW-Vaults-Count` is a holdover from Leave-Network where multiple kinds of vault data is returned. In fairness, they're all gateway vaults (as opposed to delegate vaults). This also distinguishes against ordinary vaults that might be associated with the wallet holder. I can remove the GW- prepend from the operator stake ones though.

File: src/main.lua (hunk 1 above)
Comment (arielmelendez) [reply src/main.lua]: a5edd23

File: src/main.lua (hunk 2 above)
Comment (arielmelendez) [reply src/main.lua]: a5edd23

**Pull request merged**